lib/compat/wordpress-6.1/date-settings.php

```diff
@@ -45,10 +45,10 @@ function gutenberg_update_date_settings( $scripts ) {
 				array(
 					'l10n'     => array(
 						'locale'        => get_user_locale(),
-						'months'        => array_values( $wp_locale->month ),
-						'monthsShort'   => array_values( $wp_locale->month_abbrev ),
-						'weekdays'      => array_values( $wp_locale->weekday ),
-						'weekdaysShort' => array_values( $wp_locale->weekday_abbrev ),
+						'months'        => is_array( $wp_locale->month ) ? array_values( $wp_locale->month ) : null,
```

Comment:
I don't know anything about any of this code, but if `'months'`, etc. were previously arrays, would it be better to default them to `[]` instead of `null`? Unless the JS that uses this is ok with having a null value.

Reply:
I'm not entirely sure how it works, but the client side does have a fallback/inheritance mechanism when values don't exist. I was thinking that `null` would fit with that better than `[]`